### PR TITLE
add  ip6_addresses sensor

### PR DESCRIPTION
**Pull request by h3mmy**
This is a change tied to https://github.com/home-assistant/android/issues/4198 and simply adds an entry for the new sensor

### CODE REVIEW

**Comment by dshokouhi** [comment]
Thanks for the PR, features are first included as part of the beta before an actual release is cut. Please add a beta label to this table row and we will remove it when we cut the next production release. You may refer to a previous PR for the beta label format.

File: docs/core/sensors.md [hunk 1]
```diff
@@ -273,7 +273,7 @@ The `battery_power` sensor attempts to convert microamperes to amperes however s
 
 ## Bluetooth Sensors
 ![Android](/assets/android.svg)<br />
-This Bluetooth Connection state will be the total number of connected bluetooth devices. The sensor will update as soon as the bluetooth state of the device changes. This sensor makes use of Android's [Bluetooth](https://developer.android.com/reference/android/bluetooth/package-summary?hl=en) package.
+This Bluetooth Connection state will be the total number of connected bluetooth devices. The sensor will update as soon as the bluetooth state of the device changes. This sensor makes use of Android's [Bluetooth](https://developer.android.com/reference/android/bipluetooth/package-summary?hl=en) package.
```
Comment (dshokouhi) [inline docs/core/sensors.md:276]: looks like a typo slipped in :)

Comment (h3mmy) [reply docs/core/sensors.md:276]: Whoops! I'll fix that real quick

**Review by dshokouhi — APPROVED** [review]
Looks good to me, can be merged when parent is merged

**Pull request merged**